sign-in.txt

```diff
@@ -10,8 +10,12 @@ TODO: Use the terminal and git commands to complete the PULL; MERGE; COMMIT; PUS
 
   NAME  : GitHub Username
   -----------------------------
+<<<<<<< HEAD
```

Comment:
Be sure you remove all of the Merge conflict flags from the final file.
<<<<<<<< HEAD